lib/platform.py

```diff
@@ -85,6 +85,13 @@ def have_caps(*args):
   )
   return get_caps(*args) is not None, failmsg
 
+@memoize
+def have_string_api():
+  from .common import get_media
+  has_strapi = False
+  has_strapi = get_media()._get_platform_stringapi_enable()
+  return has_strapi is not None, False
+
```

Comment:
I don't think this should be a platform cap.  The string api is not platform dependent... rather, it is middleware plugin dependent.

I envision this would be defined at the appropriate middleware util file, instead.  For example, if the option is global, in `lib/ffmpeg/util.py` we would add a method to probe for an ffmpeg option (similar to `have_ffmpeg_filter_options`).  If the option is a filter level option, then `have_ffmpeg_filter_options` can be used directly.  Then in `lib/ffmpeg/qsv/util.py` we can define `have_string_api` that simply calls that option probe with `qsv_params` as the parameter.